tox.ini

```diff
@@ -36,26 +38,28 @@ commands =
     coverage html -d {toxworkdir}/htmlcov --rcfile setup.cfg
     diff-cover --compare-branch {env:DIFF_AGAINST:origin/master} {toxworkdir}/coverage.xml
 depends =
-    py35,
     py36,
     py37,
+    py38,
+    py39,
+    py310,
 parallel_show_output = True
 
 [testenv:lint]
 description = check the code style
-basepython = python3.7
+basepython = python3.10
```

Comment:
I think we previously linted on the oldest Python we supported running with so that new syntax/types that weren't supported would be caught.

Reply:
Yeah, but I guess if tests pass it is OK. Also 3.10 will be old soon :-)

Reply:
We specify `python_version = 3.6` in our mypy_self_check.ini and run mypyc compiled tests on the oldest Python we support, so we should be fine!

Reply:
Good point @hauntsaninja !